### PR TITLE
Optimize images & improve introduction to webpack

### PR DESCRIPTION
**Pull request by addyosmani**
What's changed, or what was fixed?
- Optimized images in the webpack articles we landed today
- Improved the introduction, adding illustrations and linking up the content better

**Target Live Date:** as soon as we can manage

- [X] This has been reviewed and approved by (addyo)
- [X] I have run `gulp test` locally and all tests pass.
- [X] I have added the appropriate `type-something` label.
- [X] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/fundamentals/performance/webpack/index.md [hunk 1]
```diff
@@ -9,18 +9,33 @@ book_path: /web/fundamentals/_book.yaml
 
 {% include "web/_shared/contributors/addyosmani.html" %}
 
-Modern web applications often use a bundling tool to create a production “bundle” of files (scripts,
-stylesheets, etc.) that is optimized, minified and can be downloaded in less time by your users. In
-this article, we will walk through how to effectively optimize site resources using webpack. This
+Modern web applications often use a **bundling tool** to create a production “bundle” of files (scripts,
+stylesheets, etc.) that is [optimized](/web/fundamentals/performance/optimizing-content-efficiency/javascript-startup-optimization), [minified](/web/fundamentals/performance/optimizing-content-efficiency/optimize-encoding-and-transfer) and can be downloaded in less time by your users. In
```
Comment (petele) [inline src/content/en/fundamentals/performance/webpack/index.md]: please line break this at 80 chars

File: src/content/en/fundamentals/performance/webpack/index.md [hunk 2]
```diff
@@ -9,18 +9,33 @@ book_path: /web/fundamentals/_book.yaml
...
+**Web Performance Optimization with webpack**, we will walk through how to effectively optimize site resources using [webpack](https://webpack.js.org/). This
 can help users load and interact with your sites more quickly.
 
+<figure>
+  <img src="./webpack-logo.png" alt="webpack logo">
+</figure>
+
 webpack is one of the most popular bundling tools in use today. Taking advantage of its features for
-optimizing modern code, code-splitting scripts into critical and non-critical pieces and stripping
+optimizing modern code, [code-splitting](/web/fundamentals/performance/webpack/use-long-term-caching#lazy-loading) scripts into critical and non-critical pieces and stripping
```
Comment (petele) [inline src/content/en/fundamentals/performance/webpack/index.md]: line breaks

File: src/content/en/fundamentals/performance/webpack/index.md [hunk 3]
```diff
@@ -9,18 +9,33 @@ book_path: /web/fundamentals/_book.yaml
...
 out unused code (to name but a few optimizations) can ensure your app has a minimal network and
 processing cost.
 
-Let’s get started by looking at optimizing one of the costliest resources in a modern app –
-JavaScript.
+<figure>
+  <img src="./code-splitting.png" alt="Before and after applying JavaScript optimizations. Time-to-Interactive is improved">
```
Comment (petele) [inline src/content/en/fundamentals/performance/webpack/index.md]: line breaks

**Comment by addyosmani** [comment]
@petele Next commits should remove the optimized images (savings were up to 30% but some were much lower (5%)) and updated the lines to break at 80.

**Comment by robdodson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by addyosmani** [comment]
\o/

**Pull request merged**